tests/PHPStan/Levels/data/invalidBound.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace PHPStan\Generics\InvalidBound;
```

Comment:
I think we should drop this file and the jsons... it slows down the test suite. If it tests something that TemplateTypeDeclarationRuleTests don't, it should be moved there.